### DISCUSSION TITLE
vsce package     ERROR  Manifest missing field: version

### DISCUSSION THREAD

**Discussion by Yovenzor** [eclipse-theia/theia#13536] [Q&A]
Im trying to publish my vscode theme, I already login and everything going well, until now, I write "vsce package", and tells me this: "Manifest missing field: version":

this is the packaje.json:`{
  "name": "goosebumps-at-midnight",
  "displayName": "Goosebumps at midnight",
  "description": "This theme gives you goosebumps; Specially at midnight!",
  "publisher": "Goosebumpsatmidnight",
  "keywords": [
    "theme",
    "dark theme",
    "Goosebumps",
    "midnight",
    "yovenzor"
  ],
  "engines": {
    "vsce": "^2.15.0",
    "vscode": "^1.52.0"
  },
  "categories": [
    "Themes"
  ],
  "contributes": {
    "themes": [
      {
        "label": "Goosebumps at midnight",
        "uiTheme": "vs-dark",
        "path": "./themes/Goosebumps at midnight-color-theme.json"
      }
    ]
  },
  "dependencies": {
    "vsce": "^2.15.0"
  }
}
`

**Comment by Yovenzor** [comment]
![Screenshot from 2024-03-27 13-25-18](https://github.com/eclipse-theia/theia/assets/102180213/2635914b-8a4b-43fc-805f-a980cd5a28a2)

**Comment by msujew** [comment]
Hey @Yovenzor,

are you perhaps looking for the [vsce](https://github.com/microsoft/vscode-vsce) repo? We, as the Theia project, have nothing to do with the vsce cli tool.

Aside from that: your package.json file requires a `version` entry.

**Reply by Yovenzor** [reply]
where?

**Reply by msujew** [reply]
Something like this:

```json
{
  "name": "goosebumps-at-midnight",
  "version": "0.0.1",
  ...
}
```
